consts/lightboxOptions.js

```diff
@@ -0,0 +1,35 @@
+export const ANIMATION_DURATION_SETTINGS = {
+    fade: 250,
+    zoom: 300,
+}
+
+export const LIGHTBOX_STYLES = {
+    container: {
+        background: 'white',
+    },
+    button: {
+        filter: 'none',
+        color: 'black',
+    },
+}
+
+export const CONTROLLER_SETTINGS = {
+    closeOnBackdropClick: true,
+    closeOnPullDown: true,
+}
+
+export const CAROUSEL_SETTINGS = {
+    isFinite: true,
```

Comment:
**question:** maybe disable the endless carousel and the carousel effect in general ?